SU-SWS/stanford_basic
# 8.5.3

# READY FOR REVIEW

# Summary
- Ensure the class is always added to the mobile search area.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. check out this branch
2. view the site on mobile and verify the search area is visible.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
